src/main/kotlin/com/amazon/ionelement/impl/StructElementImpl.kt

```diff
@@ -31,7 +31,7 @@ internal class StructElementImpl(
 ) : AnyElementBase(), StructElement {
 
     override val type: ElementType get() = ElementType.STRUCT
-    override val size = allFields.size
+    override val size: Int get() = allFields.size
```

Comment:
🗺️ Both `ListElementImpl` and `SexpElementImpl` (by way of inheritance from `SeqElementImpl`) delegate to the underlying collection rather than storing the size in a field. It's necessary to delegate to the underlying collection so that we don't read the size from the collection before it is fully populated by the element loader.